app/frontend/styles/components/job-alert-feedback.scss

```diff
@@ -0,0 +1,3 @@
+#job-alert-feedback-form-comment-field {
+  height: 250px;
```

Comment:
I think you can give the form builder object a rows argument, if that's what this is trying to achieve?